### ISSUE TITLE
Disabling ScriptNo causes page-load problems

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#143]
```
What steps will reproduce the problem?

1. Click the ScriptNo icon in the toolbar
2. Click the red Disable button
3. Go to www.rottentomatoes.com

What is the expected output? What do you see instead?

I get a webpage not available error, "Error 101 (net::ERR_CONNECTION_RESET): 
The connection was reset" every time. This doesn't happen when ScriptNo is 
completely disabled (i.e., disabled in Chrome as opposed to within the 
extension). It also doesn't happen when it is fully enabled.


What version of the product are you using? On what operating system?

ScriptNo v1.0.6.2
Google Chrome Version 23.0.1271.64 m
Windows 7 64-bit with SP1


Please provide any additional information below.

That's about it.

```

Original issue reported on code.google.com by `kevinpac...@gmail.com` on 26 Nov 2012 at 3:13
- Merged into: #96

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in v1.0.6.11, which I'll be releasing either today or 
tomorrow.
```

Original comment by `andr...@gmail.com` on 1 Jan 2013 at 7:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 1 Jan 2013 at 7:58
- Changed state: **Duplicate**
